product_docs/docs/pgd/5/rel_notes/pgd_5.5.1_rel_notes.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "EDB Postgres Distributed 5.5.1 release notes"
+navTitle: "Version 5.5.1"
+---
+
+Released: 31 Mar 2024
+
+EDB Postgres Distributed version 5.5.1 is a patch release containing bug-fixes for EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.5.1. See [PGD/TPA upgrades](../upgrades/tpa_overview) for details.
+!!!
+
+
+## Bug fixes
+
+| Component | Version | Description                                                                                                                                                                                                                                                                                                                                                                   | Ticket |
+|-----------|---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------|
+| BDR       | 5.5.1   | <details><summary>Fix a data inconsistency issue with mixed version usage during upgrades</summary>Upgrading from any previous PGD version to PGD 5.5.0 may result in inconsistencies when replicating from the newer PGD 5.5.0 node to an older version PGD node. This release fixes mixed version operation to allow for smooth rolling upgrades.</details> |        |
```

Comment:
From my understanding, after reviewing the RCA, it appears that this data inconsistency issue can potentially occur in specific cases. Therefore, I am adjusting the summary and description to reflect this.

Suggest - 
Fix potential data inconsistency issues with mixed versions during a rolling upgrade 

Backward incompatible change in PGD 5.5.0 may lead to inconsistencies when replicating from a newer PGD 5.5.0 node to an older version of the PGD node, specifically during the mixed mode rolling upgrade.

This release addresses a backward compatibility issue in mixed version operation, enabling seamless rolling upgrades.

Reply:
This branch has already been merged and will require a new PR to be prepared.